docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |sync my account with other platforms |save the trouble of entering my personal details
+
+|`* * *` |user |access my data over multiple devices |do not have key in my data repeatedly
+
+|`* * *` |user |have my account logged in every time |do not have to remember one additional username and password
+
+|`* * *` |user |set reminders |do not forget what I need to do
+
+|`* * *` |user |export the list of people in my address book |store it on the computer
+
+|`* * *` |user |set events and send the event invites to any person in my contact list |calendars can sync the events
+
+|`* * *` |user |attach personal notes to people in my contact list |take notes about the person for my own reference
+
+|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else
+seeing them by accident
+
+|`* *` |user |send mass messages to a group of contacts |plan for meetings, conferences or gatherings
+
+|`* *` |user |see my groupings in a more visual display |see who are also in the group
+
+|`* *` |user |commands to be in the button form |don't have to memorise the command and it is easier to select
+
+|`* *` |user |have meta-data for my contact list |sort them according to different criteria
+
+|`* *` |user |the application should not take up too much space on my device |space for other applications
+
+|`* *` |user |the application should suggest certain fields for autocomplete |do not have to type them
+
+|`* *` |user |easily send emails to my contacts |more convenient to contact my friends
+
+|`* *` |user |update my location |contacts will know where is my current location
+
+|`* *` |user |update my status |contacts will know whether I’m busy
+
+|`* *` |user |view live stories of my friends |stay updated on what’s going on with my friends
+
+|`* *` |user |start a live chat with my friends |get an instant response from them
+
+|`* *` |user |add a profile picture |others will know how I look like
+
+|`* *` |user |change the colour scheme |make it personalized
+
+|`* *` |user |choose to require a password every time the application is opened |application stays secure
+
+|`* *` |user |access the applications quickly |be efficient with  what I do
+
+|`* *` |user |do not have to change applications to access different functionality |not troublesome for me
+
+|`* *` |user |add profiles for my friends |store their photos as well
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user |a collage of a person's profile from various social media platforms |snapshot of the person's information
+from various accounts.
+
+|`*` |user |subscribe to different channels |get more information on the outside world
+
+|`*` |user |play music in the background |feel relaxed while using the application
+
+|`*` | elderly user |application to read the contacts out loud |do not need to strain eyes reading the words
```

Comment:
avoid straining my eyes reading the words

Reply:
Done